examples/python/multiprocessing/main.py

```diff
@@ -10,6 +10,10 @@
 import rerun as rr  # pip install rerun-sdk
 
 
+# Python does not guarantee that the normal atexit-handlers will be called at the
+# termination of a multiprocessing.Process. Explicitly add the `shutdown_at_exit`
+# decorator to ensure data is flushed when the task completes.
+@rr.shutdown_at_exit
 def task(child_index: int) -> None:
     # All processes spawned with `multiprocessing` will automatically
```

Comment:
That comment (which predates the PR) doesn't make sense to me and I vote to remote it.

> All processes spawned with `multiprocessing` will automatically be assigned the same default recording_id.

No they won't, that's why we need to call `init` in all children processes too :thinking:

Reply:
They *do* have the same default recording-id... that's why calling init *works* instead of creating a new recording. But we still have to call init to create new sinks.